src/cms/templates/events/event_form.html

```diff
@@ -120,20 +120,33 @@ <h1 class="heading">
                         <span class="inline-block mb-2 font-bold">{% trans 'Status' %}:</span>
                         {{ event_translation_form.instance.get_status_display }}
                     {% endif %}
-                    <label for="{{ event_translation_form.title.id_for_label }}" class="block mb-2 mt-4 font-bold">{{ event_translation_form.title.label }}</label>
+                    <label for="{{ event_translation_form.title.id_for_label }}" 
+                        data-slugify-url="{% url 'slugify_ajax' region_slug=region.slug language_slug=language.slug model_type='event' %}{% if event_form.instance.id %}?model_id={{ event_form.instance.id }}{% endif %}"
+                        class="block mb-2 mt-4 font-bold">{{ event_translation_form.title.label }}</label>
                     {% render_field event_translation_form.title|add_error_class:"border-red-500" class="appearance-none block w-full bg-gray-200 text-xl text-gray-800 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
-                    <label for="{{ event_translation_form.slug.id_for_label }}" class="block my-2 font-bold">{{ event_translation_form.slug.label }}</label>
-                    <div class="mb-2 text-s text-gray-600">{{ event_translation_form.slug.help_text }}</div>
-                    <div id="slug-div" class="appearance-none block w-full bg-gray-200 text-xl text-gray-800 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400">
-                        {% spaceless %}
-                            <div style="display: table; white-space: nowrap;">
-                                <span style="display: table-cell;">https://integreat.app/{{ region.slug }}/{{ language.slug }}/events/</span>
-                                {% if event_translation_form.instance.ancestor_path %}
-                                    <span style="display: table-cell;">{{ event_translation_form.instance.ancestor_path }}/</span>
-                                {% endif %}
-                                <span style="display: table-cell; width: 100%;">{% render_field event_translation_form.slug|add_error_class:"slug-error" class="w-full rounded" %}</span>
-                            </div>
-                        {% endspaceless %}
+                    <div id="link-container" class="flex items-center">
```

Comment:
Could you please just add a little margin between the two form fields for events and pois?
Depending on the browser there is no/little gap between currently.
On chrome:
![Screenshot from 2021-06-21 18-06-54](https://user-images.githubusercontent.com/47010475/122794222-4ea43100-d2bc-11eb-983c-a35ce8f265ef.png)
On firefox:
![Screenshot from 2021-06-21 18-07-50](https://user-images.githubusercontent.com/47010475/122794302-6085d400-d2bc-11eb-99d7-82a050afdbee.png)

Reply:
Sure! I should have tested this with chrome, too.